Pod/Classes/OAStackViewAlignmentStrategy.m

```diff
@@ -98,25 +99,21 @@ - (NSLayoutAttribute)centerAttribute {
 }
 
 - (void)addConstraintsOnOtherAxis:(UIView*)view {
-  id arr = [self constraintsalignViewOnOtherAxis:view];
+  NSArray *arr = [self constraintsalignViewOnOtherAxis:view];
   [self.constraints addObjectsFromArray:arr];
   
   if (arr) { [self.stackView addConstraints:arr]; }
 }
 
 - (void)alignView:(UIView*)view withPreviousView:(UIView*)previousView {
-  id arr = [self constraintsAlignView:view afterPreviousView:previousView];
+  NSArray *arr = [self constraintsAlignView:view afterPreviousView:previousView];
   [self.constraints addObjectsFromArray:arr];
   
   if (arr) { [self.stackView addConstraints:arr]; }
 }
 
-- (NSMutableArray *)constraints {
-  if (!_constraints) {
-    _constraints = [@[] mutableCopy];
-  }
-  
-  return _constraints;
+- (NSMutableArray *)addedConstraints {
+  return self.constraints;
```

Comment:
return signature of this method should be of type `NSArray *`, in addition, copy should be returned not mutable instance